src/main/java/uk/gov/justice/probation/courtcasematcher/restclient/CourtCaseServiceRestHelper.java

```diff
@@ -102,10 +106,14 @@ RetryBackoffSpec buildRetrySpec(String intialMessage, BiFunction<Long, Integer,
                 .filter(EXCEPTION_RETRY_FILTER);
     }
 
-    WebClient.RequestHeadersSpec<?> get(String path) {
+    WebClient.RequestHeadersSpec<?> get(String path, Map<String, List<String>> queryParams) {
```

Comment:
I'd suggest rather than passing null queryParams to this method in the client we might be better off overloading it. i.e. 

```
WebClient.RequestHeadersSpec<?> get(String path, Map<String, List<String>> queryParams) {
   ...
}

WebClient.RequestHeadersSpec<?> get(String path){
   return get(String path, null);
}
```